gui/src/installer/view.rs

```diff
@@ -884,6 +885,131 @@ pub fn define_bitcoin<'a>(
     )
 }
 
+pub fn select_bitcoind_type<'a>(progress: (usize, usize)) -> Element<'a, Message> {
+    layout(
+        progress,
+        "Choose Bitcoin installation type",
+        Column::new().push(text(prompt::SELECT_BITCOIND_TYPE)).push(
+            Row::new()
+                .align_items(Alignment::End)
+                .spacing(20)
+                .push(
+                    Container::new(
+                        Column::new()
+                            .spacing(20)
+                            .align_items(Alignment::Center)
+                            .push(
+                                p1_regular("I want to manage my own Bitcoin node")
+                                    .style(color::GREY_3),
+                            )
+                            .push(
+                                button::secondary(None, "Select")
+                                    .width(Length::Fixed(200.0))
+                                    .on_press(Message::SelectBitcoindType(
+                                        message::SelectBitcoindTypeMsg::UseInternal(false),
+                                    )),
+                            )
+                            .align_items(Alignment::Center),
+                    )
+                    .padding(20),
+                )
+                .push(
+                    Container::new(
+                        Column::new()
+                            .spacing(20)
+                            .align_items(Alignment::Center)
+                            .push(p1_regular("Let Liana manage my node").style(color::GREY_3))
```

Comment:
nit: Could this text directly be in the button below instead of naming the button "Select"?

Reply:
I've updated this in https://github.com/wizardsardine/liana/commit/b4451e8f6332507117963ad9628ddc9a28bfee5b.